public/client.js

```diff
@@ -97,7 +97,19 @@ $(function () {
       $('#chatApprovalButtons').show();
     }
   })  
-  
 
+  // Chat Logic
+  $('#enterChatButton').click(function(){
+    $('#lobby').hide();
+    $('#chat').css('display', 'contents');
```

Comment:
Is it possible to replace line 104 with this:

$('#chat').show();

Reply:
I originally had that but $('#chat').show(); sets 'display' to 'block' so I looked it up and it said to do what I did on github.
![image](https://user-images.githubusercontent.com/28262511/99898801-8208d980-2c61-11eb-90f7-2203ad6dadcb.png)

Reply:
https://stackoverflow.com/questions/9260009/how-to-add-displayinline-block-in-a-jquery-show-function

Reply:
And block doesn't work for my ui for some reason